docs/api/guarding-policy-api.mdx

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 7
+title: Policy Guard API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The Policy Guard API allows you to manage policies at the organization level. Policy Guards help enforce centralized control over permissions and roles across specific projects included in the policy.
+For example, you can create  that enforces a specific set of permissions, as role admin can create documents, and role viewer can only read documents, across all projects included in the policy.
```

Comment:
You are missing a word/or more after "you can create...."

Reply:
The example missing some grammar and structure. 
a better way to demonstrate an example is telling the example, and then demonstrating it, like - 
We want to make sure only admins can create a document, and all other roles cannot, to achieve this we'll create a policy guard that....